### ISSUE TITLE
Do not play with BOTS! I got an e-mail from niantic.

### ISSUE THREAD

**Issue by DXjohn** [finndev/PokeBuddy#935]
![niantic](https://cloud.githubusercontent.com/assets/20984670/17618458/3f80c868-6080-11e6-9f87-c3f17150289f.png)
Here is what i get from nantic.. I were level 33 guys.
Good luck.

**Comment by Kokainium** [comment]
then u got banned or?

**Comment by DXjohn** [comment]
No, i wrote e-mail to niantic, and they post me this. I mean this is a new kind of ban.

**Comment by Kokainium** [comment]
weird

**Comment by DXjohn** [comment]
I got banned about 6 hours ago

**Comment by DXjohn** [comment]
unban with pokestops not working

**Comment by JewBagel** [comment]
you wrote to them first so idk what you showing this changes ....

**Comment by Kokainium** [comment]
sounds kinda stupid to me too..

**Comment by Kokainium** [comment]
maybe just wait ?

**Comment by Rab89** [comment]
Wait.... you emailed them and then got banned :laughing: 

GG niantic

**Comment by DXjohn** [comment]
but with this i mean that the niantic upgrade their ban system.
noo i get banned first, then i emailed them...

**Comment by Kokainium** [comment]
he was banned and emailed them to unban him if i understand it right

**Comment by Hyperhk** [comment]
yes so actually the email helps since i havent been unbanned for hours XD

**Comment by Kokainium** [comment]
i really wanna call u idiot :P u are cheating?

**Comment by JewBagel** [comment]
no where in that email does it say hes banned he pretty much ratted himself out if anything

**Comment by DXjohn** [comment]
I'm not stupid, of course, that I not wrote them I'm running bot. I wrote them something else.

**Comment by Smmw** [comment]
What have you done to get banning like this?

**Comment by DXjohn** [comment]
I let the application catch and spin pokestops, after an hour began catching Pokémon error (chatchflee) and pokestops stopped from getting ITEM and xp.

**Comment by Smmw** [comment]
what is your walking speed in the setting? I have used the bot about a half month, I open it all the day. how come this happened to you...

**Comment by Rab89** [comment]
why are you continuously attempting to bot after being banned

**Comment by JewBagel** [comment]
so out of millions of people they permabanned you doesn't make sense ive seen so far you and some other guy say youve been permabanned yet everyone else is fine you must've did something to have them watching you.. ive been botting since the day the bots started working again and nothing happened i dont get it if they really are watching there doing a shit job

**Comment by fabiosb01** [comment]
i'm receiving this message: failed to get game data from the server... anyone else???

**Comment by Kokainium** [comment]
i think the server is down :)

**Comment by JewBagel** [comment]
servers are fine i was just playing on my phone something wrong with the bot when i turned it on last time it didn't do the usual updating thing which seems odd

**Comment by Kokainium** [comment]
i dont have any pokestops and not moving

**Comment by Rab89** [comment]
@JewBagel in the smartphone app do you see pokestops and gyms fine?

**Comment by naor173** [comment]
yes sir @Rab89  just spinned and everything fine but i didnt see a pokemon on the near by but i dont think that means something

**Comment by Kokainium** [comment]
it's the bot

**Comment by Rab89** [comment]
@naor173 yep false alarm my game didnt load right lol

**Comment by JewBagel** [comment]
yeah everything loaded just fine i think something is up with the bot it needs an update

**Comment by naor173** [comment]
also the servers are not working properly that's what i think

**Comment by etowery22** [comment]
The devs turned off pokebuddy from what I understand.

**Comment by xReVoLT-iTheme** [comment]
https://github.com/finndev/PokeBuddy/issues/912

**Comment by mufclaw** [comment]
necro .7.0 is still the safest i think dont get much trouble on there ive been walking at 100km and not had 1 softban in 2 hours ive stopped now though this looking quite serious
